flow/designs/asap7/mock-array-big/Element/io.tcl

```diff
@@ -50,7 +50,7 @@ proc zip {list1 list2} {
 foreach {direction direction2 names} $assignments {
     set mirrored [zip {*}$names]
     set_io_pin_constraint -region $direction:* -pin_names [lindex $names 0]
-    set_io_pin_constraint -group -order -pin_names [lindex $names 0]
```

Comment:
@eder-matheus Why didn't this fail before this fix?

Reply:
I don't understand the question. It should not fail. You can use `-group` and `-order` together.

Reply:
OK.